vuepress/docs/next/tutorials/ecr/github-actions-workflow-from-ecg.md

```diff
@@ -0,0 +1,219 @@
+---
+sidebarDepth: 2
+---
+
+# Create a GitHub Actions Workflow From the Entando Component Generator
+
+## Overview
+With the [Entando Component Generator](../../tutorials/backend-developers/generate-microservices-and-micro-frontends.html#overview) (ECG), you can quickly create a ready-to-run Workflow file using the JHipster `ci-cd` sub-generator.
+
+This tutorial shows how to:
+1. Build a simple Continuous Integration Workflow on GitHub Actions using the ECG
+2. Modify the base file to meet requirements
+
+## Prerequisites
+For this tutorial, you will need an existing project or to generate a new one with the Entando Component Generator. Your project must be hosted on a public GitHub repository to qualify for unlimited usage. Using a private repository could lead to restrictions that are not covered here.
+
+Actions features are enabled by default without additional configuration. However, it is good practice to commit and push files remotely to validate changes.
+
+## Start generating a GitHub Action Workflow
+
+From your main project folder:
+
+1. Run `npm install` if needed
+2. Run the JHipster `ci-cd` subgenerator and select `GitHub Actions` (e.g. run `ent jhipster ci-cd` in the ent CLI) 
+3. Deselect all `tasks/integrations` to generate the default Workflow
```

Comment:
Deselect is confusing. In my test, no tasks/integrations are chosen. Instead I hit <ENTER> to take the default which includes no integrations.

Reply:
Sorry, meant to Start a Review...